tests/oracle/test-oracle.cpp

```diff
@@ -386,6 +415,69 @@ TEST_CASE("Oracle null user-defined objects in/out", "[oracle][null][type_conver
     CHECK(ind == i_null);
 }
 
+TEST_CASE ( "Oracle", "[oracle][rvalue]" )
```

Comment:
This test shouldn't Oracle-specific, should it? I.e. using rvalues should work, and be tested, with all backends AFAICS, and so this test should be in `common-tests.h`.

Reply:
I've used dual table and implicit conversion int->string. I think it's oracle-specific. But I try add a test like this into common-tests.h

Reply:
done